play/src/i18n/es-ES/camera.ts

```diff
@@ -36,6 +36,18 @@ const camera: DeepPartial<Translation["camera"]> = {
         nameTag: "Usted",
     },
     disable: "Apaga tu cámara",
+    menu: {
+        moreAction: "Más acciones",
+        closeMenu: "Cerrar el menú",
+        senPrivateMessage: "Enviar un mensaje privado (próximamente)",
+        kickoffUser: "Expulsar usuario",
+        muteAudioUser: "Silenciar audio",
+        muteAudioEveryBody: "Silenciar audio para todos",
+        muteVideoUser: "Silenciar vídeo",
+        muteVideoEveryBody: "Silenciar vídeo para todos",
+        pin: "Fijar",
+        blockOrReportUser: "Bloquear o informar de usuario",
+    },
```

Comment:
I usually never translate the text of the languages I don't speak.
I have had several feedbacks of German speakers telling us that our translations were wrong. That's because Google Translate cannot translate propertly on such a small context.

I think we should not translate those. The framework will fallback to English, and the community can provide adequate translations. They will be better at this than us.

Reply:
I use a specific IA to translate the text. I think that we should trust it 💪